code/lib/cli/src/detect.test.ts

```diff
@@ -139,6 +139,16 @@ const MOCK_FRAMEWORK_FILES: {
       },
     },
   },
+  {
+    name: ProjectType.NEXTJS,
+    files: {
+      'package.json': {
+        dependencies: {
+          next: '^9.0.0',
```

Comment:
curious: why version 9?

Reply:
Why 9 in general? Because your addon from which this is ~copy/pasted~ _based_ already supported it.

Why 9 right here? To test the version range detection.